### PR TITLE
Bonnie 1199 update patient import

### PR DESCRIPTION
**Pull request by lukoktonos**
This PR integrates the update_facilities_and_diagnoses rake task with the patient import rake task, so that it doesn't need to be run for every patient import.

NOTE: After this PR is integrated, the patient export instructions need to be edited to remove the step to update facilities and diagnoses.

Pull requests into Bonnie require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] This PR is into the correct branch.
- [x] JIRA ticket for this PR: https://jira.mitre.org/browse/BONNIE-1199
- [x] JIRA ticket links to this PR
- [x] Code diff has been done and been reviewed (it **does not** contain: additional white space, not applicable code changes, debug statements, etc.)
- [x] If UI changes have been made, google WAVE plug-in has been executed to ensure no 508 issues were introduced. N/A
- [x] Tests are included and test edge cases
- [x] Tests have been run locally and pass (remember to update Gemfile when applicable)
- [x] Code coverage has not gone down and all code touched or added is covered. 

Branch | Coverage
-- | --
master | 1566 / 1783 LOC (87.83%) covered.
bonnie-1199-update-patient-import | 1690 / 1919 LOC (88.07%) covered.

- [x] Automated regression test(s) pass

If JIRA tests were used to supplement or replace automated tests:
- [x] JIRA test links: N/A
- [x] Justification for using JIRA tests: N/A
- [x] JIRA tests have been added to sprint N/A

**Reviewer 1:** @mayerm94

Name:
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

If JIRA tests were used to supplement or replace automated tests:
- [x] JIRA tests have been run and pass N/A
- [x] You agree with the justification for use of JIRA tests or have provided input on why you disagree N/A

**Reviewer 2:**

Name:
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

If JIRA tests were used to supplement or replace automated tests:
- [x] JIRA tests have been run and pass N/A
- [x] You agree with the justification for use of JIRA tests or have provided input on why you disagree N/A

### CODE REVIEW

**Review by mayerm94 — CHANGES_REQUESTED** [review]
Should remove the QDM Measure support in the patient import and export tasks inside bonnie.rake IGNORE THIS COMMENT: CHANGES MADE IN OTHER PR INTO CQL_TESTING_OVERHAUL

**Review by mayerm94 — DISMISSED** [review]

File: .rubocop_todo.yml [hunk 1]
```diff
@@ -568,7 +568,7 @@ Metrics/MethodLength:
 
 # Offense count: 16
 Metrics/PerceivedComplexity:
-  Max: 33
+  Max: 38
```
Comment (jbradl11) [inline .rubocop_todo.yml]: I think it would be better to not modify these values and instead add the new code to the rubocop ignore file.  Thoughts?

**Review by mayerm94 — APPROVED** [review]
Latest rubocop changes make sense

**Review by jbradl11 — APPROVED** [review]
Code changes makes sense. New unit test looks good and passes.

**Comment by edeyoung** [comment]
failing due to vulnerability on master (loofah). We will be udpating.

**Pull request merged**